docker-compose.yaml

```diff
@@ -73,19 +76,19 @@ services:
           - coffeeshop-service
       command: >
         /bin/bash -c "
-          while ! curl -s http://barista-http:8082/health/ready | grep '\"BaristaReadinessCheck\",\"status\":\"UP\"' ;
+          while ! curl -sI http://barista-http:8082/health/ready ;
```

Comment:
I think I might have made this change, but given the problem you describe with Liberty's readiness check, perhaps this change should be reverted?  (same comment for the other services)

Reply:
Done.  To get it working I found I had to explicitly set the container names in the docker-compose file as it was generating names prefixed by my directory name otherwise.